app/code/Magento/Ui/Model/Export/ConvertToXml.php

```diff
@@ -158,7 +158,8 @@ public function getXmlFile()
         $stream = $this->directory->openFile($file, 'w+');
         $stream->lock();
 
-        $excel->setDataHeader($this->metadataProvider->getHeaders($component));
+        $headers = array_map(array($this, 'removeQuote'), $this->metadataProvider->getHeaders($component));
```

Comment:
Please, use the anonymous function instead of declaring the new public method.

Reply:
Hello @sidolov,

I have remove array_clean function and use anonymous function instead of separate function.
Please check/review the changes and let me know if further inputs is required from my-side.

Thanks
Nirav Patel